### PR TITLE
#2981 mobile fixes [do not merge]

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**

### CODE REVIEW

**Comment by AlexeyTuboltsev** [comment]
don't review, it's a draft!

File: app/components/dashboard/eto-list/EtoList.module.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.progress-section-layout {
+  @include media-breakpoint-up(md) {
+    grid-template: auto / repeat(4, 1fr);
+  }
+
+  @media (min-width: 1100px) {
```
Comment (drakkein) [inline app/components/dashboard/eto-list/EtoList.module.scss]: Quick question here, why we use `1100px`? Cant we just use `lg` breakpoint or `xl`? Non-blocking, I just want to understand reason.

File: app/components/dashboard/my-portfolio/MyPortfolioWidget.tsx [hunk 2]
```diff
@@ -77,7 +78,7 @@ export const MyPortfolioWidgetComponent: React.FunctionComponent<IProps> = ({
   balanceNeu,
   isIncomingPayoutAvailable,
 }) => (
-  <Panel className={cn(className, styles.panelFix)} style={style}>
+  <Panel className={cn(className, styles.panelFix)} style={style} columnSpan={EColumnSpan.TWO_COL}>
     <MyPortfolioWidgetComponentBody
       {...{
```
Comment (drakkein) [inline app/components/dashboard/my-portfolio/MyPortfolioWidget.tsx]: I know this is not yours, but this spreading looks ugly here. Can we just use regular props>

File: app/components/eto/overview/EtoOverviewThumbnail/EtoCardPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+
+import { Panel } from "../../../shared/Panel";
+
+import * as styles from "./EtoCardPanel.module.scss";
+
+interface IBaseProps {
```
Comment (drakkein) [inline app/components/eto/overview/EtoOverviewThumbnail/EtoCardPanel.tsx]: We've got `TDataTestId` interface, can we extend it within this interface?

File: app/components/eto/overview/EtoOverviewThumbnail/EtoOverviewThumbnail.tsx [hunk 4]
```diff
@@ -40,82 +39,68 @@ const defaultEmpty = "-";
...
+        {eto.company.brandName}
+      </Heading>
+
+      {etoSubState === EEtoSubState.COMING_SOON ? (
+        <p data-test-id="eto-overview-status-founders-quote" className={styles.quote}>
+          {eto.company.keyQuoteFounder}
+        </p>
+      ) : (
+        <>
+          <div className={cn(styles.groupWrapper)}>
```
Comment (drakkein) [inline app/components/eto/overview/EtoOverviewThumbnail/EtoOverviewThumbnail.tsx]: `cn` is not needed here

File: app/components/layouts/header/Header.tsx [hunk 5]
```diff
@@ -37,7 +37,7 @@ interface IDispatchProps {
 export const HeaderComponent: React.FunctionComponent<
   IExternalProps & IStateProps & IDispatchProps
 > = props => (
-  <Navbar dark className={cn(styles.bar, "flex-nowrap")}>
+  <Navbar dark className={cn(styles.bar)}>
```
Comment (drakkein) [inline app/components/layouts/header/Header.tsx]: `cn` is not needed here

File: app/components/shared/HeaderProgressStepper.tsx [hunk 6]
```diff
@@ -18,18 +17,10 @@ interface IProps {
...
-    <Row className="mt-4 mb-4 text-center">
-      <Col>
-        <h2 className={cn(styles.bold, { "text-warning": props.warning }, props.headerClassName)}>
-          {props.headerText}
-        </h2>
-        {props.descText && <p className="pt-3">{props.descText}</p>}
-      </Col>
-    </Row>
+    <ProgressStepper steps={props.steps} currentStep={props.currentStep} />
+    <h2 className={cn(styles.title, { "text-warning": props.warning }, props.headerClassName)}>
```
Comment (drakkein) [inline app/components/shared/HeaderProgressStepper.tsx]: Can we use `Heading` component here?

File: app/components/shared/HeaderProgressStepper.tsx [hunk 7]
```diff
@@ -18,18 +17,10 @@ interface IProps {
 
 export const HeaderProgressStepper: React.FunctionComponent<IProps> = props => (
```
Comment (drakkein) [inline app/components/shared/HeaderProgressStepper.tsx]: We could destroy props object here, but non-blocking

File: app/components/dashboard/eto-list/EtoList.module.scss (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/dashboard/eto-list/EtoList.module.scss]: it's just some random value at which it look good. Our breakpoints are at different widths where it doesn't look good :))
In general, this is the advantage of scoped css - we can use arbitrary values just to make it look good, without danger of breaking other stuff

File: app/components/dashboard/my-portfolio/MyPortfolioWidget.tsx (hunk 2 above)
Comment (AlexeyTuboltsev) [reply app/components/dashboard/my-portfolio/MyPortfolioWidget.tsx]: done

File: app/components/eto/overview/EtoOverviewThumbnail/EtoCardPanel.tsx (hunk 3 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/overview/EtoOverviewThumbnail/EtoCardPanel.tsx]: 👍

File: app/components/eto/overview/EtoOverviewThumbnail/EtoOverviewThumbnail.tsx (hunk 4 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/overview/EtoOverviewThumbnail/EtoOverviewThumbnail.tsx]: done

File: app/components/layouts/header/Header.tsx (hunk 5 above)
Comment (AlexeyTuboltsev) [reply app/components/layouts/header/Header.tsx]: done

File: app/components/shared/HeaderProgressStepper.tsx (hunk 6 above)
Comment (AlexeyTuboltsev) [reply app/components/shared/HeaderProgressStepper.tsx]: no, <Heading > is the H2 with green line on the left

**Comment by AlexeyTuboltsev** [comment]
all changes from this PR are going to be merged in https://github.com/Neufund/platform-frontend/pull/3033

**Comment by AlexeyTuboltsev** [comment]
closing as everything from here has been merged in https://github.com/Neufund/platform-frontend/pull/3033

**Pull request closed without merging**